### DISCUSSION TITLE
Source code location for debugging

### DISCUSSION THREAD

**Discussion by rhodain** [go-delve/delve#3153] [General]
Is there a way to point delve to a source code? I have a binary that I want to debug and I have the source code from GitHub. It checks the source in 
/usr/src/debug/podman-4.2.0-2.fc36.x86_64/_build/src/github.com/containers/podman/cmd/podman/main.go:31

Is there a way to point that to the structure I have from GitHub? In this case 

    <GIT_ROOT>/cmd/podman/main.go

or how to move it to the destination where the delve is looking. All the tutorials are showing examples with your own code. I am struggling with the postman a bit.

**Comment by aarzilli** [comment]
Not really clear what you mean, there is no difference whether you or someone else wrote the source code.

**Comment by derekparker** [comment]
Are you looking for something like path substitution? https://github.com/go-delve/delve/tree/master/Documentation/cli#config

**Reply by rhodain** [reply]
Yes, that was exactly what I was looking for! Thanks!
